Userland/Utilities/fortune.cpp

```diff
@@ -78,6 +90,10 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
 
     Optional<bool> force_color;
 
+    bool wait_before_termination = false;
+    int const minimum_secs_to_wait = 6;
+    int const chars_per_second = 20;
```

Comment:
These could even be `constexpr` (and unsigned).

Reply:
Done.